jobset-generator/default.nix

```diff
@@ -0,0 +1,11 @@
+{ pkgs }:
+pkgs.poetry2nix.mkPoetryApplication {
+  projectDir = ./src;
+
+  meta = with pkgs.lib; {
+    homepage = "https://github.com/DeterminateSystems/hydra-github-project";
+    description = "Declaratively generate jobsets for all a project's branches and PRs.";
+    license = licenses.mit;
+    maintainers = teams.DeterminateSystems;
+  };
+}
```

Comment:
Unrelated to this, but there's an empty `README.rst` at `jobset-generator/src/README.rst` (which is the file just below this in the GH file view) -- should that be removed?